spec/models/snap_application_spec.rb

```diff
@@ -183,4 +183,33 @@
       end
     end
   end
+
+  describe "#drive_status" do
+    context "when there are drive attempts and no errors" do
+      it "returns :drive_none" do
+        snap_application = create(:snap_application)
+
+        expect(snap_application.drive_status).to eq :drive_none
+      end
+    end
+
+    context "when there are drive attempts and no errors" do
+      it "returns :drive_in_flight" do
+        driven_application = create(:driver_application)
```

Comment:
the naming in this spec is a bit confusing -- a driven application is an instance of drive application and driven application has a snap application. 

Maybe it should be `completed_driver_application` or something like that instead?